spanner/spanner_snippets/spanner/integration_test.go

```diff
@@ -380,8 +382,20 @@ func runSample(t *testing.T, f sampleFunc, dbName, errMsg string) string {
 
 func runBackupSample(t *testing.T, f backupSampleFunc, dbName, backupID, errMsg string) string {
 	var b bytes.Buffer
-	if err := f(&b, dbName, backupID); err != nil {
-		t.Errorf("%s: %v", errMsg, err)
+	attempts := 0
```

Comment:
Please use `testutil.Retry`.

Reply:
Good point, I didn't know about the existence of that one.